tests/plugins/test_koji_import.py

```diff
@@ -176,11 +176,16 @@ def is_string_type(obj):
 
 
 class BuildInfo(object):
-    def __init__(self, value, valid=True):
-        if valid:
-            self.build = BuildResponse({'metadata': {'annotations': {'help_file': value}}})
-        else:
-            self.build = BuildResponse({'metadata': {'annotations': {}}})
+    def __init__(self, help_file=None, help_valid=True, media_types=None):
+        annotations = {}
+        if media_types is None:
```

Comment:
Just curious, but why not `if not media_types`? Are we intentionally handling an empty list differently?